File: LatPrak3_spekPc (JAVA)/Processor.java
class Processor{
	//atribut
	private String name;
	private int price;

	//constructor
	Processor(){
	}

	//get & set untuk atribut name
	public void setName(String name){
		this.name = name;
	}
		
	public String getName(){
		return this.name;
	}

	//get & set untuk atribut price 
	public void setPrice(int price){
		this.price = price;
	}
		
	public int getPrice(){
		return this.price;
	}
}

File: LatPrak3_spekPc/Processor.cpp
#include <string>

using namespace std;

class Processor{
	private:
	//atribut dalam class processor
		string name;
		int price;

	public:
		//constructor
		Processor(){
		}

		Processor(string name, int price){
			this->name = name;
			this->price = price;
		}


		//prosedur dan fungsi untuk get & set dari atribut name
		void setName(string name){
			this->name = name;
		}
		
		string getName(){
			return this->name;
		}

		//prosedur dan fungsi untuk get & set dari atribut price
		void setPrice(int price){
			this->price = price;
		}
		
		int getPrice(){
			return this->price;
		}

		
	//destructor
	~Processor(){
	}
};